@shared/api/helpers/soroban.ts

```diff
@@ -1,17 +1,16 @@
 import BigNumber from "bignumber.js";
-import { xdr, Address } from "soroban-client";
+import * as SorobanClient from "soroban-client";
 
 import { I128 } from "./xdr";
 
 /* eslint-disable */
 
 export const accountIdentifier = (account: string) =>
-  new Address(account).toScVal();
+  new SorobanClient.Address(account).toScVal();
 
 // How do we decode these in a more generic way?
-export const decodei128 = (scVal: Buffer) => {
-  const value = xdr.ScVal.fromXDR(scVal);
-
+export const decodei128 = (xdr: string) => {
+  const value = SorobanClient.xdr.ScVal.fromXDR(xdr, "base64");
```

Comment:
this took me soooo long to figure out. We have to now pass `fromXDR` the plain xdr instead of a buffer, which makes sense. It just was not apparent from the changes in the latest client release